.coderabbit.yaml

```
@@ -0,0 +1,13 @@
+language: "ko-KR"
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)